tests/examples/pacer/nef/s3/mdb_multi_2.json

```diff
@@ -0,0 +1,54 @@
+{
+  "appellate": false,
+  "contains_attachments": false,
+  "court_id": "mdb",
+  "dockets": [
+    {
+      "case_name": "Laura J. Margulies, Chapter 7 Trustee for Botanica v. Rodas",
+      "date_filed": null,
+      "docket_entries": [
+        {
+          "date_filed": "2024-11-22",
+          "description": "Adversary Case 0:24-ap-16 Closed. (Horning, Kelly)",
+          "document_number": null,
+          "document_url": null,
+          "pacer_case_id": "780665",
+          "pacer_doc_id": null,
+          "pacer_magic_num": null,
+          "pacer_seq_no": null,
+          "short_description": "Close Adversary Case"
```

Comment:
Subject in the email is
`"Multiple Cases "Close Adversary Case" - AP -"`

Should be reduced to "Close Adversary Case". It doesn't follow the usual clean up process, that's why I am using the early return
![image](https://github.com/user-attachments/assets/c225c38e-736f-4612-9e48-e0b23f01befc)